docs/guides/mysql/pitr/restic/archiver.md

```diff
@@ -0,0 +1,459 @@
+    selector:
+      matchLabels:
+        archiver: "true"
+  retentionPolicy:
+    name: mysql-retention-policy
+    namespace: demo
+  encryptionSecret:
+    name: "encrypt-secret"
+    namespace: "demo"
+  fullBackup:
+    driver: "Restic"
+    scheduler:
+      successfulJobsHistoryLimit: 1
+      failedJobsHistoryLimit: 1
+      schedule: "0 0 * * *"
+    sessionHistoryLimit: 2
+  manifestBackup:
+    scheduler:
+      successfulJobsHistoryLimit: 1
+      failedJobsHistoryLimit: 1
+      schedule: "0 0 * * *"
+    sessionHistoryLimit: 2
+  backupStorage:
+    ref:
+      name: "storage"
+      namespace: "demo"
+```
+
+### EncryptionSecret
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: encrypt-secret
+  namespace: demo
+stringData:
+  RESTIC_PASSWORD: "changeit"
+```
+
+```bash 
+ $ kubectl create -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/pitr/restic/yamls/mysqlarchiver.yaml
+ mysqlarchiver.archiver.kubedb.com/mysqlarchiver-sample created
+ 
+ $ kubectl create -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/pitr/restic/yamls/encryptionSecret.yaml
+ secret/encrypt-secret created
+```
+
+# Deploy MySQL
+We are now ready with the setup for continuous MySQL archiving. We will deploy a MySQL object that references the MySQL archiver object.
+
+```yaml
+apiVersion: kubedb.com/v1
+kind: MySQL
+metadata:
+  name: mysql
+  namespace: demo
+  labels:
+    archiver: "true"
+spec:
+  version: "8.2.0"
+  replicas: 3
+  topology:
+    mode: GroupReplication
+  storageType: Durable
+  storage:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  archiver:
+    ref:
+      name: mysqlarchiver-sample
+      namespace: demo
+  deletionPolicy: WipeOut
+```
+
+```bash 
+ $ kubectl create -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/pitr/restic/yamls/mysql.yaml
+ mysql.kubedb.com/mysql created
+ ```
+
+```bash
+$ kubectl get pod -n demo
+NAME                                                              READY   STATUS      RESTARTS   AGE
+mysql-0                                                           2/2     Running     0          15m
+mysql-1                                                           2/2     Running     0          15m
+mysql-2                                                           2/2     Running     0          15m
+mysql-archiver-full-backup-1733120326-4n8nd                       0/1     Completed   0          10m
+mysql-archiver-manifest-backup-1733120326-9gw4f                   0/1     Completed   0          10m
+mysql-sidekick                                                    1/1     Running     0          10m
+retention-policy-mysql-archiver-full-backup-1733120326-7rx9t      0/1     Completed   0          9m31s
+retention-policy-mysql-archiver-manifest-backup-1733120326l79mb   0/1     Completed   0          9m56s
+```
```

Comment:
we can mention some lines of how these pod will created or overall backup process